src/pages/Settings/PaymentSettings.vue

```diff
@@ -0,0 +1,339 @@
+<template>
+	<www-page class="payments" :gray-background="true">
+		<the-my-kiva-secondary-menu slot="secondary" />
+		<div class="title-area">
+			<div class="row column">
+				<h1>
+					Payment Methods
+				</h1>
+			</div>
+		</div>
+		<div class="row">
+			<kv-settings-card class="columns small-12 large-8" title="Credit Card Settings">
+				<template v-slot:icon>
+					<kv-icon
+						class="icon"
+						title="Payment"
+						name="unknown-card"
+					/>
+				</template>
+				<template v-slot:content>
+					<form class="payment-settings-default-form" @submit.prevent>
+						<fieldset>
+							<legend v-if="hasSavedPaymentMethods">
+								Select your default card
+							</legend>
+							<legend v-else>
+								There are no cards saved to this account
+							</legend>
+							<div v-for="(paymentMethod, index) in savedPaymentMethods"
+								:key="index"
+								class="payment-settings-default-form__cc-wrapper"
+							>
+								<kv-radio
+									class="payment-settings-default-form__radio"
+									:id="`creditCard-${index}`"
+									:radio-value="paymentMethod.nonce"
+									v-model="selectedDefaultCardNonce"
+									:checked="selectedDefaultCardNonce === paymentMethod.nonce"
+								>
+									<img class="payment-settings-default-form__cc-icon"
+										:src="paymentMethod.imageUrl"
+									> {{ paymentMethod.description }}
+								</kv-radio>
+								<kv-button
+									@click.native.prevent="showLightbox(paymentMethod)"
+									class="payment-settings-default-form__remove-btn text-link text-link--red"
+								>
+									Remove
+								</kv-button>
+							</div>
+						</fieldset>
+						<fieldset class="payment-settings-default-form__save-button-wrapper">
+							<kv-button
+								class="smaller"
+								v-if="!isProcessing"
+								@click.native="savePaymentSettings"
+								:disabled="!isChanged || $v.$invalid"
+							>
+								Save Settings
+							</kv-button>
+							<kv-button class="smaller" v-else>
```

Comment:
Another aside for the future, having a `<kv-button loading>` prop would be sweet too.

Reply:
Agreed, this code is repeated in many places